ui/src/app/machines/views/MachineDetails/MachineNetwork/NetworkFields/NetworkFields.tsx

```diff
@@ -26,17 +26,50 @@ export type NetworkValues = {
   vlan: NetworkInterface["vlan_id"];
 };
 
+/*
+
+
+ */
+
+/**
+ * Formik values eventually resolve to the correct types,
+ * meanwile we need to force the subnet value to be a number.
+ */
+const toNumber = (value: string | number): number | undefined => {
```

Comment:
as this is specifically for formik and because it can return `undefined` maybe a better name for this function is `toFormikNumber`.

Reply:
Yep, good idea.